core/lib/zksync_core/src/api_server/web3/tests/mod.rs

```diff
@@ -13,6 +13,7 @@ use zksync_dal::{transactions_dal::L2TxSubmissionResult, ConnectionPool, Storage
 use zksync_health_check::CheckHealth;
 use zksync_types::{
     api,
+    api::BlockId,
```

Comment:
Micro-nit: API server code generally uses explicit `api::` prefix for all types from the corresponding module to distinguish them from the domain types (some of which have the same name). So, in this case, it could make sense to follow this convention and use `api::BlockId`.